internal/stringify/fmt_test.go

```diff
@@ -0,0 +1,99 @@
+package stringify
+
+import (
+	"fmt"
+	"io"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+type aStringer struct {
+	v any
+}
+
+func (a aStringer) String() string {
+	return fmt.Sprintf("%v", a.v)
+}
+
+type notAStringer struct {
+	v any
+}
+
+var _ Concealer = &aConcealer{}
+
+type aConcealer struct {
+	v any
+}
+
+func (a aConcealer) Conceal() string                { return "***" }
+func (a aConcealer) Format(fs fmt.State, verb rune) { io.WriteString(fs, "***") }
+func (a aConcealer) PlainString() string            { return fmt.Sprintf("%v", a.v) }
+
```

Comment:
Could you please add tests for `Normalize` too?

Reply:
done!